crates/bevy_math/src/primitives/dim3.rs

```diff
@@ -214,6 +215,80 @@ impl InfinitePlane3d {
 
         (Self { normal }, translation)
     }
+
+    /// Computes the shortest distance between a plane transformed with the given `isometry` and a
+    /// `point`.
+    #[inline]
+    pub fn signed_distance(&self, isometry: Isometry3d, point: Vec3) -> f32 {
```

Comment:
Unless I'm misreading the function, the output seems a bit counter-intuitive.
I'd expect the distance being positive in the half-space into which the normal points.

I'm not saying it's wrong, but I believe this should be noted in the doc.

Reply:
Yeah kind of makes sense. It was just convenient for the projection but should also just be a sign flipping operation there.